rvip/src/main.cpp

```diff
@@ -0,0 +1,60 @@
+/*  Copyright (C) 2019 by Bey Hao Yun <beyhy@artc.a-star.edu.sg>
+
+    Permission to use, copy, modify, and/or distribute this software for any purpose with or without fee is hereby granted.
+
+  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL
+  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT,
+  OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
+  NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+// General Includes
+#include <pcl/exceptions.h>
+
+// Includes for ROS
+#include "ros/ros.h"
+
+// Includes for Point Cloud Library and ROS PCL
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+#include <pcl_ros/point_cloud.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <pcl_conversions/pcl_conversions.h>
+
+// For Modularity and Debugging
+#include "rvip/rvip_main.h"
+
+int main(int argc, char **argv)
+{
+        // Initializing ROS node
+        ros::init(argc, argv, "rvip_0.0.2");
```

Comment:
I don't believe `rvip_0.0.2` is a valid ROS node name.

Reply:
Fixed in [`03346e4`](https://github.com/ros-industrial/rvip/pull/1/commits/03346e46c254dc0026d30a3787c272ba316832fc). 

However, initializing node with name `rvip_0.0.2` incurrs no **ros::InvalidNameException**. Renaming is done, regardless, to improve intuitive understanding of rvip architecture.